packages/cojson-storage/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "cojson-storage",
+  "version": "0.8.35",
```

Comment:
Not sure what should I state here for a new package

Reply:
You made the right call to set the same version as the other packages.

I think it makes easier to manage dependencies this way.